sessions/react-data-fetching/demo-start/README.md

```diff
@@ -0,0 +1,56 @@
+# React Data Fetching: Jokes
```

Comment:
As we have already talked about, adapt the README.md files to the correct content of a demo.

Reply:
Update: adapted the readme files myself.